Add script to remove unnecessary files from Projects folders.

davidbaines opened the pull request:
Add script to remove unnecessary files from Projects folders. 
Requires --confirm-delete and --delete-subfolders to delete all the unnecessary files without confirmation.
Writes a csv file with the actions taken. The --dry-run option can be specified to list the files and folders that would be deleted.

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/sillsdev/silnlp/562)
<!-- Reviewable:end -->

mshannon-sil reviewed (changes requested):
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/silnlp/562)* status: all files reviewed, 1 unresolved discussion (waiting on @davidbaines)

___
*[`silnlp/common/clean_projects.py` line 1 at r1](https://reviewable.io/reviews/sillsdev/silnlp/562#-O9kX76UDWsO8g_1rEnL:-O9kX76UDWsO8g_1rEnM:bhn2qj4) ([raw file](https://github.com/sillsdev/silnlp/blob/0d5a5652b87ec4a8210ec41f45335a9ed60415f4/silnlp/common/clean_projects.py#L1)):*
> ```Python
> import argparse
> ```

I would move this `clean_projects.py` script to the `scripts` folder in the repo, since that's where the other executable script `/scripts/clean_s3.py` that cleans up files in the S3 bucket is located.


<!-- Sent from Reviewable.io -->

davidbaines commented:
Good idea. I've moved the clean_projects.py script from the common to the scripts folder.
Updated .gitignore to ignore any report files that are created by the script.

mshannon-sil reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/silnlp/562#-:-O9pSKzv4KVI5RD3Uzqu:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @davidbaines)



<!-- Sent from Reviewable.io -->
